ConcernsCaseWork/ConcernsCaseWork.Tests/Pages/Case/Management/Concern/IndexPageModelTests.cs

```diff
@@ -50,13 +53,9 @@ public async Task WhenOnGetAsync_ReturnPage()
 				.ReturnsAsync(trustDetailsModel);
 			mockRatingModelService.Setup(r => r.GetRatingsModel())
 				.ReturnsAsync(ratingsModel);
-			mockTypeModelService.Setup(t => t.GetTypeModel())
-				.ReturnsAsync(typeModel);
-			
-			var pageModel = SetupIndexPageModel(mockCaseModelService.Object, mockTrustModelService.Object,
-				mockTypeModelService.Object, mockRecordModelService.Object, mockRatingModelService.Object, mockMeansOfReferralModelService.Object, mockLogger.Object);
 
-			pageModel.Request.Headers.Add("Referer", "https://returnto/thispage");
```

Comment:
What was this doing previously? Is it removed because the code is now redundant?

Reply:
Hi blake,

From what I can see the code was always using the referrer to decide where to go back to

But when you go into the management pages, the back button should always go back to case management, so I made it a static link instead